invenio/base/bundles.py

```diff
@@ -133,11 +134,23 @@
 
 admin = Bundle(
     "js/admin.js",
+    "vendors/admin-lte/dist/js/app.min.js",
     output="admin.js",
     filters=RequireJSFilter(exclude=[jquery]),
     weight=50
 )
 
+admin_styles = Bundle(
+    "vendors/admin-lte/dist/css/AdminLTE.min.css",
+    "vendors/admin-lte/dist/css/skins/{0}.min.css".format(current_app.config.get("ADMIN_UI_SKIN")),
+    output="admin-lte.css",
```

Comment:
Please rename it to `admin.css` so we can change the skin without renaming the bundle. Thanks